ci/travis/build-docker-images.py

```diff
@@ -29,7 +29,12 @@
         "https://hub.docker.com/repository/docker/rayproject/ray-ml")
 }
 
-PY_MATRIX = {"-py36": "3.6.12", "-py37": "3.7.7", "-py38": "3.8.5"}
+PY_MATRIX = {
+    "-py36": "3.6.13",
+    "-py37": "3.7.10",
+    "-py38": "3.8.10",
+    "-py39": "3.9.5"
+}
```

Comment:
What's the main reason for bumping the Minor versions here?

Reply:
I wonder if there are important security fixes included in the latest minor versions, that users would like to have?

Reply:
Probably, but maybe do this in a separate PR--just in case something breaks with the new version.

Reply:
Ok.